.gitmodules

```diff
@@ -10,6 +10,6 @@
 	path = test/cbmc/litani
 	url = https://github.com/awslabs/aws-build-accumulator
 	update = none
-[submodule "source/dependency/3rdparty/http_parser"]
-	path = source/dependency/3rdparty/http_parser
-	url = https://github.com/nodejs/http-parser.git
+[submodule "source/dependency/3rdparty/llhttp"]
+	path = source/dependency/3rdparty/llhttp
+	url = https://github.com/nodejs/llhttp.git
```

Comment:
llhttp is a required dependency of coreHTTP without which the code cannot be used. What is the reasoning behind this suggestion?

Reply:
@aggarg It's so that you don't end up with multiple copies of a given library in a project. It's helpful to have dependencies as submodules for testing purposes, but it's usually best to leave it to the customer to determine which version set they want to use.

It's unlikely that this llhttp will be a dependency of other libraries, so it's probably okay in this case.